test/internal/kubernetes/utils_create.go

```diff
@@ -454,6 +456,24 @@ func initCNSScenarioVars() (map[CNSScenario]map[corev1.OSName]cnsDetails, error)
 				installIPMasqAgent: false,
 			},
 		},
+		EnvInstallCNSNodeSubnet: {
+			corev1.Linux: {
+				daemonsetPath:          cnsLinuxDaemonSetPath,
+				labelSelector:          cnsLinuxLabelSelector,
+				rolePath:               cnsRolePath,
+				roleBindingPath:        cnsRoleBindingPath,
+				clusterRolePath:        cnsClusterRolePath,
+				clusterRoleBindingPath: cnsClusterRoleBindingPath,
+				serviceAccountPath:     cnsServiceAccountPath,
+				initContainerArgs: []string{
+					"deploy",
+					"azure-ipam", "-o", "/opt/cni/bin/azure-ipam",
+				},
+				initContainerName:  initContainerNameIPAM,
+				configMapPath:      cnsNodeSubnetLinuxConfigMapPath,
+				installIPMasqAgent: true,
```

Comment:
Do we need IpMasq?

Reply:
Yes. We need the secondary IPs to be SNATed to the primary.